docs/az/docs/features.md

```diff
@@ -0,0 +1,203 @@
+# Xüsusiyyətlər
+
+## FastAPI xüsusiyyətləri
+
+**FastAPI** sizə aşağıdakıları təmin edir:
+
+### Açıq standartlara əsaslanır
+
+* API yaratmaq işlərində <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank"><strong>OpenAPI </strong></a> buna <abbr title="also known as: endpoints, routes">path </abbr> <abbr title=" HTTP metodları olaraq bilinən, POST, GET, PUT, DELETE">əməliyyatları, </abbr>parametrləri, body tələbi, təhlükəsizlik kim şeylər daxildir.
+* Avtomatik olaraq data modelinin <a href="http://json-schema.org/" class="external-link" target="_blank"><strong>JSON Schema </strong></a> ile birlikdə sənədləşməsi(OpenAPI özü onsuz JSON Schema əsasındadır).
+* Diqqətli iş nəticəsində yuxarıda göstərilən standartlara uyğun bir framework yaratdıq. Standartları tortun üzərinə sonradan əlavə edilən çiyələk kimi görmədik.
+* Bu həmçinin bir çox dildə istifadə oluna biləcək **client code generator** istifadəsinə də icazə verir.
+
+### Avtomatik sənədləşmə
+
+OpenAPI standartlarına əsaslanan bir framework olaraq, developerlərin birdən çox seçimi var, defolt olaraq gələn 2 fərqli interaktiv API sənədləşməsi və veb istifadəçi interfeysi var.
+
+* <a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank"><strong>Swagger UI </strong></a> sizə interaktiv olaraq brauzerdən API-ya çağırış etməyə və sınaqdan keçirməyə imkan verir.
+
+![Swagger UI interaction](https://fastapi.tiangolo.com/img/index/index-03-swagger-02.png)
+
+* <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank"><strong>ReDoc </strong></a> ilə alternativ API sənədləşməsi.
+
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-06-redoc-02.png)
+
+### Sadəcə müasir Python
+
+Tamamilə standartlar **Python 3.8**'nın type hintlərinə əsaslanır (Pydantic'in sayəsində). Yeni bir sintaksis öyrənməyə ehtiyyac yoxdur. Sadəcə müasir Python.
```

Comment:
"Tamamilə standartlar **Python 3.8**'nın type hintlərinə əsaslanır (Pydantic'in sayəsində)." -> "Tamamilə standart **Python 3.6** tip məsləhətlərinə əsaslanır (Pydantic sayəsində)."

Reply:
Type hint sözünü azərbaycan dilinə çevirmək nə qədər normaldı, sən sözün tip məsləhətləri sözünü azərabaycanca başa düşürsən?